### PR TITLE
DOCS-1059 Release Notes (4.3.6 and 5.6)

### PR DESCRIPTION
**Pull request by djw-m**
Signed-off-by: Dj Walker-Morgan <dj.walker-morgan@enterprisedb.com>

### CODE REVIEW

File: product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml [hunk 1]
```diff
@@ -0,0 +1,281 @@
+product: EDB Postgres Distributed
+version: 5.6.0
+date: 15 October 2024
+intro: |
+ EDB Postgres Distributed 5.6.0 includes a number of enhancements and bug fixes.
+ Please add more text here.
+highlights: |
+ - Streaming Transaction support with SDW.
```
Comment (PJMODOS) [inline product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml:8]: Not sure if this is highlight tbh, given we have one user of SDW, but we don't ave mch highlights, so dunno, but certainly not the top one

File: product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+ - GROUP COMMIT and SYNCHRONOUS COMMIT support DEGRADE ON.
```
Comment (PJMODOS) [inline product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml:9]: Might merge the, the next one (origin group) and the improvement in the sync commit behavior around deadlocks with writer (that I also commented on) into one big highlight talking about improvements in commit scopes maybe?

Comment (PJMODOS) [reply product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml:9]: Also if we count the new usage stats (bdr.stat_commit_scope), and current info about backends using them (bdr.stat_commit_scope_state)

File: product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml [hunk 3]
```diff
@@ -0,0 +1,281 @@
+product: EDB Postgres Distributed
```
[the remark this answers is not in the document]
Comment (PJMODOS) [reply product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml:1]: Also nothing about compatibility improvements (making notes for myself) commits:
0774b74efcd0c66cf40490c704b5f2f57059eb95
fc40449448f69f9bb25c113fd98689ebed97684d
aa7d1a10ceef9ebb80b32116e4c40cb98eb637ed
d4d75420d9cd668cd29ed68536a936f8583201c7
98eb69f841d0a6a92bc2c27a89cd71d705b54ee0
03c6302ddef9e8b52c1a10b5fc7e5d25815e0ba3

File: product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml (hunk 2 above)
Comment (PJMODOS) [reply product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml:9]: And the ability to use parent group's commit scope (which is what makes ORIGIN_GROUP appealing)

File: product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+  severity: High
+  impact: High
+- relnote: Detect and resolve deadlocks between synchronous replication wait-for-disconnected sessions and replication writer.
+  component: BDR
+  component_version: 5.6.0
+  details: |
+    This will cancel synchronous replication wait on disconnected sessions if it deadlocks against replication, preventing deadlocks on failovers when using synchronous replication. This only affects commit scopes, not synchronous replication configured via `synchronous_standby_names`.
+  jira: BDR-5445, BDR-5445, BDR-4104
+  addresses: ""
+  type: Bug Fix
```
[the remark this answers is not in the document]
Comment (djw-m) [reply product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml:53]: If it was meant to behave as PG sync rep, then it's probably a bug fix. If it wasn't and it now does, then it's an enhancement?

File: product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml (hunk 2 above)
Comment (djw-m) [reply product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml:9]: I think thats everything covered.

File: product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml (hunk 4 above)
[the remark this answers is not in the document]
Comment (djw-m) [reply product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml:53]: Marked as an enhancement

File: product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (PJMODOS) [reply product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml:1]: Added

File: product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml [hunk 5]
```diff
@@ -0,0 +1,411 @@
...
+intro: |
+ EDB Postgres Distributed 5.6.0 includes a number of enhancements and bug fixes.
+ Please add more text here.
+highlights: |
+ - Streaming Transaction support with SDW.
+ - GROUP COMMIT and SYNCHRONOUS COMMIT support DEGRADE ON.
+ - Improved monitoring
+ - ORIGIN\_GROUP support to simplify commit scope creation.
+ - Optimized Toplogy support for Subscriber-only groups and nodes. (preview)
+ - Improved Postgres compatibility? (exclusion constraints, REINDEX replication, createrole_self_grant, column reference in DEFAULT expressions, CREATE SCHEMA AUTHORIZATION ...)
```
[the remark this answers is not in the document]
Comment (djw-m) [reply product_docs/docs/pgd/5.6/rel_notes/src/relnote_5.6.0.yml:13]: Updated compatibility to compliance.

**Review by PJMODOS — APPROVED** [review]
I think this is looking okay

**Pull request merged**